generic/BRGeneric.c

```diff
@@ -204,6 +209,23 @@ genTransferSerialize (BRGenericTransfer transfer, size_t *bytesCount) {
     return transfer->handlers.getSerialization (transfer->ref, bytesCount);
 }
 
+static size_t
+genTransferSetHash (const void *transferPtr) {
+    BRGenericTransfer transfer = (BRGenericTransfer) transferPtr;
+    return genTransferGetHash (transfer).value.u32[0];
```

Comment:
Isn't this suppose to be "SetHash"

Reply:
It is actually correct - but about the worst possible naming...

Reply:
Addressed issue, renamed functions.